packages/purpose-process/src/services/purposeService.ts

```diff
@@ -1666,10 +1666,13 @@ async function activatePurposeLogic({
     updatedAt: new Date(),
     firstActivationAt: new Date(),
   };
-
+  const unsuspendedByConsumerPurpose: Purpose =
+    fromState === purposeVersionState.waitingForApproval
+      ? { ...purpose.data, suspendedByConsumer: false }
```

Comment:
since the final state is always `Active`, I would say that also `suspendedByProducer` should be set to `false`